backend/cmd/migrate/migrate.go

```diff
@@ -187,6 +207,12 @@ func main() {
 	logger, _ := zap.NewDevelopment()
 	logger = logger.WithOptions(zap.AddStacktrace(zap.FatalLevel + 1))
 
+	// Check that both namespace and migration dir are either empty or filled.
+	if (f.Namespace == "") != (f.MigrationDir == "") {
```

Comment:
When `MigrationsTable == postgres.DefaultMigrationsTable` and if `len(f.MigrationDir) > 0` we should probably stop the user from accidentally corrupting their database, no?

Also this check can go elsewhere i just thought of it while reading this line.

Reply:
I guess this case should be sufficient

Reply:
the namespace is appended to `{$DefaultMigrationsTable}_` so it's not possible to accidentally override the default migrations table

Reply:
I see now cool.